tests/tests/multiple_files_tests.robot

```diff
@@ -27,32 +24,34 @@ Extract two files one by one test
     [Tags]    multiple    performance
 
     # Adding and Extracting First File
-
-    Extract File    ${file_4mb}    ${file_4mb}[path]
-    Check If Zip File Is Not Empty    ${DOWNLOAD_PATH}    ${file_4mb}
+    ${test_file}=    Select random file
+    Extract File    ${test_file}    ${test_file}[path]
+    Check If Zip File Is Not Empty    ${DOWNLOAD_PATH}    ${test_file}
     # Adding and Extracting Second File
-    Rename Downloaded Zip File Name    ${DOWNLOAD_PATH}    ${file_4mb}
+    Rename Downloaded Zip File Name    ${DOWNLOAD_PATH}    ${test_file}
     Clean Input List
-    Extract File    ${file_4mb}    ${file_4mb}[path]
-    Check If Zip File Is Not Empty    ${DOWNLOAD_PATH}    ${file_4mb}
+    ${test_file}=    Select random file
+    Extract File    ${test_file}    ${test_file}[path]
+    Check If Zip File Is Not Empty    ${DOWNLOAD_PATH}    ${test_file}
 
 Extract multiple files test
     [Documentation]    Extract file consisting of multiple files
     [Tags]    multiple    performance
     Log To Console    Extracting file consisting of multiple files
-    Extract Multiple Files    ${multi_part_4mb}    ${DOWNLOAD_PATH}
-    Check If Zip File Is Not Empty    ${DOWNLOAD_PATH}    ${multi_part_4mb}
+    ${test_file}    Set Variable    ${Multi_part_4MB}
```

Comment:
as above

Reply:
Agreed with Krzysztof that 'Set variable' topic will be handle by separate PR.